src/com/lftechnology/task/dec31/DistanceManager.java

```diff
@@ -0,0 +1,207 @@
+package com.lftechnology.task.dec31;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+/**
+ * Q2. Create a class called Distance that includes two private data members
+ * feet (type int) and inches (type float). Use a method setData to set values
+ * to instance variables. Provide a method display that displays the feet and
+ * inches, a member function addDistance for adding two distances, and a member
+ * function compareDistance for comparing two distances. Implement your program
+ * in Java.
+ * 
+ * @author Sanjib Maharjan
+ * 
+ */
+public class DistanceManager {
+	public static void main(String[] args) {
+		Scanner userInput = new Scanner(System.in);
+		Distance distanceInput1 = new Distance();
+		Distance distanceInput2 = new Distance();
+		System.out.println("Input first value for distance");
+		setDistanceInput(distanceInput1, userInput);
+		System.out.println("Input second value for distance");
+		setDistanceInput(distanceInput2, userInput);
+		userInput.close();
```

Comment:
So, Why are you avoiding the previous pull request comments here. I thought you guys were using try catch for such things.